changes/380.feature.md

```diff
@@ -0,0 +1,5 @@
+Use `.get()` instead of `.build()` for `OptionBuilder`.
```

Comment:
The change shouldn't be classified as a `feature` (filename), but should be classified as `misc`.